.github/workflows/test-models.yml

```diff
@@ -123,7 +123,7 @@ jobs:
         echo "VULKAN SDK PATH with setup: $VULKAN_SDK"
         echo $PATH
         pip list | grep -E "torch|iree"
-        pytest --ci --ci_sha=${SHORT_SHA} --local_tank_cache="/Volumes/builder/anush/shark_cache" tank/test_models.py -k vulkan --update_tank
+        pytest -s --ci --ci_sha=${SHORT_SHA} --local_tank_cache="/Volumes/builder/anush/shark_cache" tank/test_models.py -k vulkan --update_tank
```

Comment:
this path `/Volumes/builder/anush/shark_cache` is only on the mac builder

Reply:
Right-- the other builders are pointing to `/data/anush/shark_cache/` . Is this incorrect? I thought I had these configured correctly.

Reply:
follow-up - weren't these all supposed to be pointing towards an external volume located at `data/anush/` for linux and `/Volumes/builder/anush/` for the mac builder?
Let me know if this isn't the case so I can fix it.